README.rst

```diff
@@ -8,141 +8,225 @@ PHP PDO driver for Snowflake
+   .. code-block:: bash
 
-.. note::
+       export PHP_HOME=$WORKSPACE/install-php-$SF_PHP_VERSION
 
-    If you don't have :code:`pdo_snowflake.so`, build it following the instruction below.
+   Otherwise, if the :code:`phpize` executable is in :code:`/usr/bin`, run the following command:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-    cp cacert.pem /etc/php/7.2/fpm/conf.d/
+       export PHP_HOME=/usr
 
-Add a config file :code:`/etc/php/7.2/fpm/conf.d/20-pdo_snowflake.ini` including the following contents to the PHP config directory.
+#. Clone the :code:`pdo_snowflake` repository, and run the script to build the driver:
 
-.. code-block:: text
+   .. code-block:: bash
 
-    extension=pdo_snowflake.so
-    pdo_snowflake.cacert=/etc/php/7.2/fpm/conf.d/cacert.pem
-    # pdo_snowflake.logdir=/tmp     # location of log directory
-    # pdo_snowflake.loglevel=DEBUG  # log level
+       git clone https://github.com/snowflakedb/pdo_snowflake.git
+       cd pdo_snowflake
+       ./scripts/build_pdo_snowflake.sh
 
-Restart Nginx and PHP-FPM services. For example:
+#. Run the following command to verify that the driver can be loaded into memory successfully:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-    service nginx restart
-    service php7.2-fpm restart
+       $PHP_HOME/bin/php -dextension=modules/pdo_snowflake.so -m | grep pdo_snowflake
 
-Ensure :code:`phpinfo()` function return the output including :code:`pdo_snowflake`.
+Building the Driver on Windows
+------------------------------
 
-.. code-block:: bash
+#. Clone the :code:`pdo_snowflake` repository:
 
-    curl http://localhost/test.php | grep -E "(pdo|json|snowflake)"
+   .. code-block:: batch
 
-.. note::
+       git clone https://github.com/snowflakedb/pdo_snowflake.git
+       cd pdo_snowflake
 
-    We have not finalized what package would be the best for binary distribution. So far I'm trying to get :code:`pecl` account but have not got one yet. Any suggestion is welcome.
+#. Run the script to download the PHP SDK:
 
-Windows
--------
+   .. code-block:: batch
 
-There are two required files for you to copy:
+       setup_php_sdk.bat <arch> <build> <visual studio version> <path to PHP SDK>
 
-- php_pdo_snowflake.dll
-- cacert.pem
+   where:
 
-Copy :code:`php_pdo_snowflake.dll` to the same location as :code:`php_pdo.dll` where all PHP extensions reside (usually the :code:`ext` folder in your PHP installation).
+   - :code:`<arch>` is your CPU architecture (:code:`x64` or :code:`x86`).
+   - :code:`<build>` is the type of binary that you want to build (:code:`Release` or :code:`Debug`).
+   - :code:`<visual studio version>` is the version of Visual Studio that you are using (:code:`VS14` or :code:`VS15`).
+   - :code:`<path to PHP SDK>` is the path to the directory where the PHP SDK should be downloaded.
+     **Do not create this directory.** The script creates this directory for you when downloading the PHP SDK.
```

Comment:
Is this correct? The script won't clone the repository if the directory already exists?

Reply:
Yea so if that directory exists then we assume that the PHP git repo has been downloaded previously and so we just skip re-downloading the git repo to save time. Users should have the script create the folder the first time and then skip the download if it has already happened

Reply:
Yes, that's the behavior of git clone command.